qemu/tests/hugepage_specify_node.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from virttest import env_process
+from virttest import error_context
+from virttest import utils_misc
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Qemu allocate hugepage from node.
+    Steps:
+    1) System setup hugepage on host.
+    2) Mount this hugepage to /mnt/kvm_hugepage.
+    3) Boot guest only allocate hugepage from node 0.
+    4) Check the hugepage used only from node 0.
+    :params test: QEMU test object.
+    :params params: Dictionary with the test parameters.
+    :params env: Dictionary with test environment.
+    """
+    error_context.context("Get host numa topological structure.", logging.info)
+    host_numa_node = utils_misc.NumaInfo()
+    node_list = host_numa_node.online_nodes
+    error_context.base_context("Specify qemu process only allocate HugePages "
+                               "from node 0.", logging.info)
+    vm = "vm_bind_to_0"
```

Comment:
The `vm` being a string object causes ambiguity

Reply:
correct